hack/build-images.sh

```diff
@@ -7,7 +7,7 @@ set -o pipefail
 STARTTIME=$(date +%s)
 source_root=$(dirname "${0}")/..
 
-prefix="openshift/origin-"
+prefix="docker.io/openshift/origin-"
```

Comment:
Possible this was miscommunication - see origin and how it's structured.  This probably belongs in release.